### PR TITLE
Scala Compiler Options minor copy edits

### PR DESCRIPTION
**Pull request by sephlietz**

### CODE REVIEW

File: _overviews/compiler-options/index.md [hunk 1]
```diff
@@ -27,21 +27,21 @@ discourse: true
...
 
-Nowadays, most people are not running scalac from the command line.
-Instead, they use sbt, IDE, and other tools as their interface to the compiler.
-Therefore they may not even have scalac installed, and won't think to do `man scalac`.
+Nowadays, most people are not running `scalac` from the command line.
+Instead, they use sbt, an IDE, and other tools as their interface to the compiler.
+Therefore they may not even have `scalac` installed, and won't think to do `man scalac`.
 
-This page come to the rescue for the people to find...
+This page comes to the rescue for the people to find&hellip;
```
[the remark this answers is not in the document]
Comment (sephlietz) [reply _overviews/compiler-options/index.md]: I agree with you but I'd rather leave larger copy edits to another PR.

**Comment by martijnhoekstra** [comment]
Looks good to me, also without the further suggestions. Drone seems stuck. I think @jvican is best positioned to get that unstuck?

**Comment by heathermiller** [comment]
Hrm, @martijnhoekstra you're right. Don't know what's up with Drone and it's not letting me restart the build. In any case, nothing in @sephlietz's PR looks like it would cause any issues. So, merging and will fix if any issues arise post-merge. 

THANK YOU @sephlietz! ❤️

**Comment by jvican** [comment]
CI is fixed :wink:, sbt/zinc had some agents blocked on some pending PRs and there were no agents left for this repository.

**Pull request merged**